src/javascript/Edit/Edit.adapter.js

```diff
@@ -1,20 +1,24 @@
 import dayjs from '~/date.config';
 import {getDynamicFieldSets, getFields} from '~/EditPanel/EditPanel.utils';
 import {resolveSelectorType} from '~/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/SelectorTypes.utils';
-import {adaptSections} from '~/FormDefinitions/FormData.adapter';
+import {adaptSections, getFieldValuesFromDefaultValues} from '~/FormDefinitions/FormData.adapter';
 
 const getInitialValues = (nodeData, sections) => {
     // Retrieve dynamic fieldSets
     const dynamicFieldSets = getDynamicFieldSets(sections);
 
     // Retrieve fields and the return object contains the field name as the key and the field value as the value
-    const fields = getFields(sections)
+    const nodeValues = getFields(sections)
         .reduce((result, field) => ({...result, ...getFieldValues(field, nodeData)}), {});
 
+    // Get default values for not enabled mixins
+    const extendsMixinFieldsDefaultValues = getFields(sections, undefined, fieldset => fieldset.dynamic && !fieldset.activated)
```

Comment:
```suggestion
    const extendsMixinFieldsDefaultValues = getFields(sections)
```
getFields doesn't accept a function as a third parameter

Reply:
maybe you wanted a filter ?

Reply:
getFields does accept a 3rnd params that is a fct: 
 * @param {array} sections    array object contains sections
 * @param {string} sectionName string value refer to the section name	 * @param {string} sectionName string value refer to the section name
 * @param {string} fieldSetFilter optional fieldset filter

fieldSetFilter